### PR TITLE
BLD: raster speed test

### PR DESCRIPTION
**Pull request by ArianaFreitag**

### CODE REVIEW

File: src/rasterizer/model.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+learning_rate = 1e-4
+# t = number of back prop
+for t in range(1):
+    # start time for effeciency of model
+    start = time.time()
+
+    y_pred = model(x)
+    end = time.time()
+    # duration model ran in seconds
+    print(end - start)
```
Comment (eigenfoo) [inline src/rasterizer/model.py]: This currently times how long the _whole model_ is taking to forward propagate and back propagate. Really what we'd like to see is a layer-by-layer breakdown of how long each layer is taking to forwardprop and backprop. Idk how you would do that, though 😛

Comment (ArianaFreitag) [reply src/rasterizer/model.py]: do we really care about the other dummy layers?

Comment (eigenfoo) [reply src/rasterizer/model.py]: We care about the speed of the rasterizer as compared to "reasonably sized" dummy generators and discriminators - and we can only know what a "reasonable size" is when we time it.

Also reminded me of a quote:
> If you use a ruler to measure a table, you may also be using the table to measure the ruler.

Comment (ccurro) [reply src/rasterizer/model.py]: You don't need a complicated layer by layer break down. Simply time the generator, then time the discriminator (with say a random input) and sum them

**Pull request closed without merging**